src/ResourceManager/Resources/Commands.Resources/help/New-AzureRmADServicePrincipal.md

```diff
@@ -585,6 +585,8 @@ Parameters: ApplicationObject (ByValue)
 
```

Comment:
@maddieclayton we may want to add an example to both cmdlets for the new scenario and show the output containing the password